irohad/main/application.cpp

```diff
@@ -514,9 +514,3 @@ Irohad::RunResult Irohad::run() {
             return e;
           });
 }
-
-Irohad::~Irohad() {
```

Comment:
Why is `freeConnections()` function call no longer needed?

Reply:
It's called in Storage destructor, no need to do it manually from here.